### ISSUE TITLE
update marker Lat Lng

### ISSUE THREAD

**Issue by DanialV** [johnpryan/flutter_map#188]
I couldn't find a way to update marker Lat Lng is there any way that can update marker location?

**Comment by vblago** [comment]
You can remove this marker and add a new one. Marker is added by changing the list of marker layer options and state changing after that.

**Comment by ganisback** [comment]
Yes you have to remove and recreate it, this widget does not provide the update method

**Comment by DanialV** [comment]
@vblago I think it is not a best practice to remove and recreate a marker just for a simple position change

**Comment by rubberbird** [comment]
It would be good to add all the animate marker leaflet functions

**Comment by rubberbird** [comment]
> You can remove this marker and add a new one. Marker is added by changing the list of marker layer options and state changing after that.

When I marker.add(new Marker) and then set state, I get the map reloading again. Surely there is a way to move marker location without refreshing the map?
